createDirectories.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DIRECTORY="$(git rev-parse --show-toplevel)"
+
+if [[ $# -lt 1 ]]; then
+  echo "Usage: bash createDirectories.sh <programming language>" >&2
+  exit 1
+fi
+
+programmingLanguage="$1"
+rsync --archive --include='*/' --exclude='*' "${DIRECTORY}/Python/" \
```

Comment:
oh i just forgot a crucial detail: git never stores empty directories! so you might want to create empty files afterwards:

```bash
find "${DIRECTORY}/${programmingLanguage}" -mindepth 2 -maxdepth 2 -exec touch {}/empty.txt \;
```

Reply:
Oh that's right 😢

Reply:
i just got merge privileges back! do you want to fix this before it's merged?

Reply:
let me do that